index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link 
+    rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css" 
+    integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A==" 
+    crossorigin="anonymous" referrerpolicy="no-referrer" 
+    />
+</head>
+<body>
+    <main>
+        <header>
+            <!--Header Navbar-->
+            <nav>
+                <div class="container">
+                    <div class="logo">
+                        <h4>My Logo</h4>
+                    </div>
```

Comment:
- [ ] Kindly remove your logo from the nav tag, only the hamburger menu should've wrapped with the nav tag. Also, add a link tag `<a>` to your logo, it's used for semantics and also to improve user experience.